service-registry/src/main/java/org/apache/servicecomb/serviceregistry/consumer/MicroserviceVersions.java

```diff
@@ -109,8 +122,6 @@ public void pullInstances() {
         DefinitionConst.VERSION_RULE_ALL,
         revision);
     if (microserviceInstances == null) {
```

Comment:
pull failed or setInstance failed, will never try again in watch mode except notified by another event.
i remember it's a bug in old version
now, it will happened too?

Reply:
in dynamic schema loading, when server not started, pull fail due to service not exists, at this time this not notification will given by sc and we will fetch create MicroserviceVersions again when call the method. check logic in MicroserviceVersions.